src/controllers/commands/CommandController.cpp

```diff
@@ -2829,6 +2829,54 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
+    this->registerCommand("/banid", [formatBanTimeoutError](
```

Comment:
Open to other suggestions for the command name

Reply:
this name is fine for me